ndmg/scripts/ndmg_dwi_pipeline.py

```diff
@@ -144,21 +226,22 @@ def main():
                         labels of regions of interest in atlas space")
     parser.add_argument("-c", "--clean", action="store_true", default=False,
                         help="Whether or not to delete intemediates")
-    parser.add_argument("-f", "--fmt", default='edgelist',
-                        choices=['gpickle', 'graphml', 'edgelist'],
+    parser.add_argument("-f", "--fmt", action="store", default='csv',
                         help="Determines graph output format")
+    parser.add_argument("-b", "--big", action="store_true", default=False,
+                        help="whether or not to produce voxelwise big graph")
     result = parser.parse_args()
 
     # Create output directory
-    cmd = "mkdir -p {} {}/tmp".format(result.outdir, result.outdir)
-    print("Creating output directory: {}".format(result.outdir))
-    print("Creating output temp directory: {}/tmp".format(result.outdir))
+    cmd = "mkdir -p " + result.outdir + " " + result.outdir + "/tmp"
+    print("Creating output directory: " + result.outdir)
+    print("Creating output temp directory: " + result.outdir + "/tmp")
```

Comment:
What is the reason for switching away from `.format()` syntax here?

Reply:
fixed back